File: account-svc/src/main/java/club/hosppy/account/model/AccountSecret.java
package club.hosppy.account.model;

import lombok.*;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
@Entity
@Table(name = "account")
public class AccountSecret {

    @Id
    private String id;

    private String passwordHash;

    private String email;

    private boolean confirmedAndActive;
}

File: src/main/kotlin/club/hosppy/account/model/AccountSecret.kt
package club.hosppy.account.model

import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "account")
class AccountSecret {
    @Id
    var id: Int? = null
    var passwordHash: String = ""
    var email: String = ""
    var confirmedAndActive = false
}